nv2a-trace.py

```diff
@@ -66,7 +72,12 @@ def _wait_for_stable_push_buffer_state(
         # So we pause the pusher again to validate our state
         xbox_helper.pause_fifo_pusher()
 
-        time.sleep(1.0)
+        time.sleep(0.1)
```

Comment:
Why is this possible?

I usually fine-tuned most of these waits after running into issues with various games.
I assume I had my reasons for choosing such high wait time.

Ideally none of these waits would exist, instead, there'd be code on the Xbox which actively monitors the GPU state (and also controls and guards it more reliably, against changes by the game).